app/services/cocina/to_fedora/descriptive/event.rb

```diff
@@ -74,15 +74,19 @@ def note(note)
           xml.send(note.type, note.value, attributes)
         end
 
+        # the only contributors legal for MODS are ones with role publisher
         def contributor(contributor)
+          return unless contributor_role_publisher?(contributor)
+
           attributes = {}
           name = contributor.name.first
           if name.valueLanguage
             attributes[:lang] = name.valueLanguage.code
             attributes[:script] = name.valueLanguage.valueScript.code
             attributes[:transliteration] = name.standard.value if name.standard
           end
-          xml.send(contributor.role.first.value, name.value, attributes)
+
+          xml.send('publisher', name.value, attributes)
```

Comment:
why do send if we know the name of the method?
```suggestion
          xml.publisher(name.value, attributes)
```

Reply:
good point.  fixing.